init/main.js

```diff
@@ -1,15 +1,15 @@
-const execa = require('execa')
-const { red } = require('chalk')
+import execa from 'execa'
+import chalk from 'chalk'
```

Comment:
chalk version `5.0.0` has es module support so you could update and use named imports

Reply:
Once all our repositories have migrated to pure ES modules, I plan on updating our shared Renovate configuration, removing all the exceptions related to pure ES modules.
This will create PRs to upgrade each dependency, as a second separate step after the actual migration.